src/pages/Landingpage.tsx

```diff
@@ -1,33 +1,73 @@
-import {
-  Button,
-  TextField,
```

Comment:
Das gehört eigentlich allesin eine ander PR. Aber jetztwo dus schon gemacht hast, können wirs auch so lassen.
Bitte bedenke, dass das Mergen einer PR zu einem einzelenen Commit wird! Hier sind ganz viele Sachen drin, die nichtsmiteinander zutun haben. Also ab jetzt bitte kleinere PRs :)

Reply:
Ja voll, macht Sinn. War mir nicht genau sicher wie man das macht und habe de fakto einfach immer meine arbeit des sprints in eine Pull Request gepackt.